addon/lib/shooter.js

```diff
@@ -377,11 +392,12 @@ const ShotContext = Class({
       } catch (e) {
         // Ignore... it's just a best effort to cancel the state
       }
-      this.interactiveWorker.destroy();
+      setTimeout(() => {
+        this.interactiveWorker.destroy();
+        this.interactiveWorker = null;
+      }, 1000);
```

Comment:
There was a race between the worker finishing and destroy destroying it... I guess there should be a fixme in here to send a message to the worker, which sends a message back when it is finished, and then destroy gets called after that... But this seemed to be enough for the worker to tear down the selection handles before destroy gets called. (Of course there's still a risk that it will race. But much less.)